src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -367,13 +367,13 @@ export class DBTCoreProjectIntegration
     const queryThread = this.executionInfrastructure.createPythonBridge(
       this.projectRoot.fsPath,
     );
-    await this.createPythonDbtProject(queryThread);
-    await queryThread.ex`project.init_project()`;
     return new QueryExecution(
       async () => {
         queryThread.kill(2);
       },
       async () => {
+        await this.createPythonDbtProject(queryThread);
+        await queryThread.ex`project.init_project()`;
```

Comment:
why this change?

Reply:
Doesn't matter as this is not affecting main thread.